pkg/kubernetes/client.js

```diff
@@ -680,6 +697,34 @@ define([
         });
 
         /*
+         * add_watcher:
+         * @type: The type of watch to add
+         *
+         * Returns a Promise that if successfull will
+         * be a KubernetesWatch object for the given type.
+         * The same object will be available at self.watches[type]
+         */
+        self.add_watcher = function add_watcher (type) {
```

Comment:
No you're right about this. self.include(type) makes more sense, especially if you have to access client.watches afterwards (as you do in the tests). Sorry for the bum steer.